app/src/main/java/com/android/unio/model/firestore/transform/Serialization.kt

```diff
@@ -41,7 +41,7 @@ fun AssociationRepositoryFirestore.Companion.serialize(association: Association)
  * @param members Members to map.
  * @return Map of user UIDs to role UIDs.
  */
-fun mapUsersToRoles(members: List<Member>): Map<String, String> {
+private fun mapUsersToRoles(members: List<Member>): Map<String, String> {
```

Comment:
You may be right that the function should not be private here, however it means modifying the tests accordingly - as you did. For example in #262 I modify the Role object which means I should also remodify the tests ? Even if that's not too long, I think it's better to centralize everything that is just duplicated three times in the project. 
Wouldn't it be okay to put it in a tool file so that conceptually we can leave it `public` ?